pip/commands/install.py

```diff
@@ -251,12 +251,12 @@ def run(self, options, args):
                     progress_bar=options.progress_bar,
                 )
 
-                self.populate_requirement_set(
-                    requirement_set, args, options, finder, session, self.name,
-                    wheel_cache
-                )
-
                 try:
+                    self.populate_requirement_set(
```

Comment:
I don't think that this change is needed - populate_requirement_set moving inside the try.

Am I missing something?